dev/com.ibm.websphere.javaee.servlet.sip.1.1/src/javax/servlet/sip/SipApplicationSession.java

```diff
@@ -252,6 +252,27 @@ Object getSession(java.lang.String id,
     Iterator getSessions(String protocol) throws IllegalStateException, NullPointerException, IllegalArgumentException;
     
     
+    
+    /**
+     * Returns an <code>Iterator</code> over the "protocol" session objects
+     * associated of the specified protocol associated with this application
+     * session. If the specified protocol is not supported, an empty
+     * <code>Iterator</code> is returned.
+     * If "SIP" is specified the result will be an <code>Iterator</code>
+     * over the set of {@link SipSession} objects belonging to this application
+     * session. For "HTTP" the result will be a list of
+     * <code>javax.servlet.http.HttpSession</code> objects.
+     * @param protocol a string identifying the protocol name, e.g. "SIP"
+     * @param boolean representing if SIP sessions should be created if none exist
+     * @return <code>Iterator</code> over protocol sessions of the
+     *      specified protocol  
+     * @throws IllegalStateException - if this application session is not valid
+     * @throws NullPointerException - if the protocol is null 
+     * @throws IllegalArgumentException - if the protocol is not understood by container.
+     * 
+     */
+    Iterator getSessions(String string, boolean b) throws IllegalStateException, NullPointerException, IllegalArgumentException; ;
```

Comment:
Just for readability, could you update the params names to `String protocol, boolean create`, same as in the implementation.

Reply:
parameter names changed both in SipApplicationSession and SipApplicationSessionImpl